src/lib/utils/loadstor.h

```diff
@@ -55,6 +56,9 @@
 
 namespace Botan {
 
+static_assert(std::endian::native == std::endian::big || std::endian::native == std::endian::little,
+              "Mixed endian systems are not supported");
```

Comment:
Super nit: should that go into a more central header? `target_info.h` or one of its friends?

Reply:
I am trying hard to keep header dependencies to a minimum for compilation time reasons, and I think it's likely this `static_assert` will never fire unless someone as a joke decides to cross compile for VAX. So leaving it here seems fine to me, especially since this is where the assumption re endian dualism are being made.